src/components/util/AutocompleteInput/AutocompleteList.tsx

```diff
@@ -70,7 +70,9 @@ export const AutocompleteList = React.forwardRef<HTMLDivElement, AutocompleteLis
 
 		const list = React.useMemo(() => {
 			if (!value || alwaysShow) return items;
-			return items.filter(item => item.toLowerCase().includes(value.toLowerCase()));
+			return items.filter(
+				item => item.toLowerCase().includes(value.toLowerCase()) && value !== item,
```

Comment:
doesnt it solve the same issue you tried to solve by adding a 'closedOnClick' prop onAutocompleteInput, if so it should be deleted

Reply:
I checked on the 4657 branch - this fix was for a different problem. We got rid of AutocompleteAnchor, so that it wouldn't re-select all the last sessions. 

This fix is made, to show all the autocomplete options. Because before it when we redact a Bubble, not all options showed up.